common/tests/test_cache_managers.py

```
@@ -0,0 +1,518 @@
+        self.assertIsInstance(choices, list)
+        self.assertIsInstance(qs_json, str)
+
+    def test_build_form_choices_data_structure(self):
+        mgr = self._make_manager(choice_data=[{"color": "red"}])
+        result = mgr._build_form_choices_data()
+        self.assertIn("choices", result)
+        self.assertIn("queryset", result)
+        # choices should be list of (value, display) tuples
+        choices = result["choices"]
+        self.assertEqual(len(choices), 1)
+        value, display = choices[0]
```

Ruff (0.15.10)

[warning] Unpacked variable `display` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)